frontend/src/components/design-system/style/BackgroundComponent.vue

```diff
@@ -21,126 +29,46 @@ const currentEmbed = inject(EmbedKey);
-        <div class="blue-rounded-spot"></div>
-        <div class="red-rounded-spot"></div>
-        <div class="white-rounded-spot"></div>
-        <div class="gray-rounded-spot"></div>
-      </template>
-      <template v-else>
-        <div class="red-rounded-spot-single"></div>
-        <div class="blue-rounded-spot-single"></div>
+    <div class="has-background">
+      <template v-if="props.hasBlobs">
+        <div class="blobs">
+          <img src="@/assets/background/blue-blob.svg" class="blue-blob"/>
+          <img src="@/assets/background/red-blob.svg" class="red-blob"/>
+        </div>
       </template>
       <slot />
     </div>
   </template>
 </template>
 
 <style scoped lang="scss">
-.background {
+.has-background {
   display: grid;
-  background-attachment: fixed;
+  width: 100%;
   height: 100%;
-
-  .blue-spot,
-  .red-spot,
-  .blue-rounded-spot,
-  .red-rounded-spot,
-  .white-rounded-spot,
-  .gray-rounded-spot,
-  .red-rounded-spot-single,
-  .blue-rounded-spot-single {
-    position: fixed;
-    pointer-events: none;
-    z-index: -1;
-    filter: blur(35px);
-  }
-
-  > .blue-spot,
-  > .red-spot {
-    display: none;
-  }
-
-  .blue-rounded-spot {
-    background-image: url(@/assets/background/round-blue-spot.svg);
-    width: 200px;
-    height: 231px;
-    inset: auto -41px 56px auto;
-  }
-  .red-rounded-spot {
-    background-image: url(@/assets/background/round-red-spot.svg);
-    width: 162px;
-    height: 162px;
-    inset: 82px auto auto 16px;
-  }
-  .white-rounded-spot {
-    background-image: url(@/assets/background/round-white-spot.svg);
-    width: 213px;
-    height: 213px;
-    inset: -35px -27px auto auto;
-  }
-  .gray-rounded-spot {
-    background-image: url(@/assets/background/round-gray-spot.svg);
-    width: 162px;
-    height: 162px;
-    inset: auto auto 56px -33px;
-  }
-  .red-rounded-spot-single {
-    background-image: url(@/assets/background/round-red-spot-single.svg);
-    width: 442.37px;
-    height: 462.75px;
-    inset: 229px -143px auto auto;
-    transform: rotate(30deg);
-  }
-
-  .blue-rounded-spot-single {
-    background-image: url(@/assets/background/round-blue-spot-single.svg);
-    width: 407.71px;
-    height: 470.91px;
-    inset: 76px 134px auto auto;
-    transform: rotate(30deg);
-  }
 }
-@media screen and (min-width: 768px) {
-  .background {
-    display: grid;
-    background-repeat: no-repeat;
-    background-color: var(--color-neutral-bg);
-    background-image: url(@/assets/background/check.svg),
-      url(@/assets/background/times.svg);
-    background-position: left -49px top 112px, right -49px top 289px;
 
-    .blue-rounded-spot,
-    .red-rounded-spot,
-    .white-rounded-spot,
-    .gray-rounded-spot {
-      display: none;
-    }
+.blobs {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
 
-    > .blue-spot,
-    > .red-spot {
-      display: block;
-    }
+  .blue-blob,
+  .red-blob {
+    height: v-bind('props.blobsHeight');
```

Comment:
Tohle by mělo fungovat i bez `props.`:

```suggestion
    height: v-bind('blobsHeight');
```